src/mkdocstrings_handlers/python/templates/material/language.html

```diff
@@ -1,5 +1,9 @@
 <!-- Import translations for given language and fallback -->
 
+{% if locale is not defined %}
```

Comment:
As you said in the main thread, it's weird that `locale` would not be defined here since you import with context :thinking:

Reply:
In the tests the templates are rendered without supplying `locale` 😅

Reply:
Ah yes of course! Well the tests should be updated then instead of the language templates. Do you think you can do that? I can take a look if needed.

Reply:
OK, i'll look into it